subsys/bluetooth/services/fast_pair/fp_storage/fp_storage_ak.c

```diff
@@ -474,6 +474,13 @@ static int fp_storage_ak_reset(void)
 	int err;
 	bool was_enabled = is_enabled;
 
+	if (was_enabled) {
+		err = fp_storage_ak_uninit();
```

Comment:
Is it done to ensure that module is in uninitialized state after `fp_storage_ak_reset` returns with error?

If we want to apply it, then we should consider to also add it for other `fp_storage` modules (to ensure consistency).

Reply:
Yes. It's also cleaner to call uninit if we want to call init after the reset.